test/WebJobs.Script.Tests/WorkerFunctionMetadataProviderTests.cs

```diff
@@ -291,5 +291,32 @@ static void AssertFunction(FunctionMetadata function)
             AssertFunction(function1);
             AssertFunction(function2);
         }
+
+        [Fact]
+        public void ValidateBindings_MasksSensitiveDataInBindings()
+        {
+            var functionMetadata = new FunctionMetadata();
+            List<string> rawBindings =
+            [
+                """{"type": "queueTrigger","name": "myQueueItem","direction": "in","queueName": "test-input-node","connection": "DefaultEndpointsProtocol=https;AccountName=a;AccountKey=b/c==;EndpointSuffix=core.windows.net"}""",
```

Comment:
Add a test with queue name (or any other field) as `"2024-04-29T11:35:00+08:00"`. You will see that the datetime is parsed to some other value. This won't happen if we use 

`new JsonSerializerSettings { DateParseHandling = DateParseHandling.None }`

Reply:
see CreateJObjectWithSanitizedPropertyValue_StringInput_DateTimeWithTimezoneOffset_RemainsUnchanged test